code/__HELPERS/roundend.dm

```diff
@@ -507,6 +513,27 @@
 		station_vault += current_acc.account_balance
 		if(!mr_moneybags || mr_moneybags.account_balance < current_acc.account_balance)
 			mr_moneybags = current_acc
+
+	parts += "<span class='big servradio'>SERVICE DEPARTMENT STATISTICS:</span></div>"
+	parts += "The bar earned [bar_served] customers.<br>"
+	parts += "The restaurant earned [restaurant_served] customers.<br>"
+	parts += "In total, they earned [tourist_income] credits!<br>"
+	log_econ("Roundend service income: [tourist_income] credits.")
+	switch(tourist_income)
+		if(0)
+			parts += "<span class='redtext'>Service did not earn any credits...</span></div>"
+		if(1 to 1000)
+			parts += "<span class='redtext'>Come on service, surely you can do better than that.</span></div>"
+			award_service(/datum/award/achievement/jobs/service_bad)
+		if(1001 to 2000)
+			parts += "<span class='greentext'>Centcom is satisfied with service's job today.</span></div>"
+			award_service(/datum/award/achievement/jobs/service_okay)
+		else
+			parts += "<span class='reallybig greentext'>Centcom is incredibly impressed with service today! What a team!</span></div>"
+			award_service(/datum/award/achievement/jobs/service_good)
```

Comment:
you should unlock lower grades too

Reply:
also should these really unlock for both if only one did it?

Reply:
Botanist is also getting these gradings, this achievement is team based. Earning more than 2000 credits anyway is pretty damn difficult without teamwork. But it's cool because we have individual achievement scores for the servers too